addons/account/views/account_move_views.xml

```diff
@@ -628,36 +629,34 @@
                     <field name="date"/>
                     <field name="state"/>
                     <templates>
-                        <t t-name="kanban-box">
-                            <div t-attf-class="oe_kanban_global_click">
-                                <div class="row mb4">
-                                    <div class="col-6 o_kanban_record_headings">
-                                        <strong>
-                                            <span>
-                                                <field name="partner_id" invisible="not partner_id" readonly="state != 'draft'" />
-                                                <field name="journal_id" invisible="partner_id" />
-                                            </span>
-                                        </strong>
-                                    </div>
-                                    <div class="col-6 text-end">
-                                        <strong><i class="fa fa-clock-o" aria-label="Date" role="img" title="Date"/> <t t-out="record.date.value"/></strong>
-                                    </div>
+                        <t t-name="kanban-card">
+                            <div class="row mb4">
+                                <div class="col-6 o_kanban_record_headings">
+                                    <strong>
+                                        <span>
+                                            <field name="partner_id" invisible="not partner_id" readonly="state != 'draft'" />
+                                            <field name="journal_id" invisible="partner_id" />
```

Comment:
some fields aren't necessary to be defined above the card as they appear in the card (it's the case for journal_id and partner_id, at least)